hwp_encoder/Beaglebone_Encoder_DAQ.c

```diff
@@ -283,12 +292,17 @@ int main(int argc, char **argv) {
         }
         // Send encoder data if the buffer is full
         if(encd_ind == ENCODER_PACKETS_TO_SEND) {
+            for (int i = 0; i < ENCODER_PACKETS_TO_SEND; i++){
+                encoder_to_send[i].packet_count = encd_pkt_cnt++;
+            }
             sendto(sockfd, (struct EncoderInfo *) encoder_to_send, sizeof(encoder_to_send), 
                    MSG_CONFIRM, (const struct sockaddr *) &servaddr, sizeof(servaddr));
             encd_ind = 0;
         }
 	// Send IRIG data if the buffer is full
         if(irig_ind == IRIG_PACKETS_TO_SEND) {
+            irig_pkt_cnt += 1;
+            irig_to_send[irig_ind].packet_count = irig_pkt_cnt;
```

Comment:
I wonder why we increment packet_count differently in encoder and irig?
Something is wrong with encoder_packet, because i is always 0?

Reply:
Good point, I think this is wrong for the irig packet counter, but I have not been checking that in my tests.